C#/Essentials/Getting Started/TryParse.md

```diff
@@ -0,0 +1,58 @@
+# TryParse
+author: VhatAmI
```

Comment:
This has to be your Enki username, so make sure it's a case sensitive exact string match.